pkg/controllers/verifier_controller.go

```diff
@@ -76,13 +73,7 @@ func (r *VerifierReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
-	namespace, err := getCertStoreNamespace(req.Namespace)
```

Comment:
Is this a breaking change? Didn't the certificate store support specification across different namespaces. This controller is for the new cluster verifier however since it uses same name as namespaced verifier, then wouldn't there be confusion if naemspace is not appended after upgrade?